src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -166,144 +332,177 @@ def reinit_data(self):
-                for entry in transceiver_info]
-        else:
-            self.transceiver_entries = []
-
-        # update cache with initial data
-        for transceiver_entry in self.transceiver_entries:
-            # extract interface name
-            interface = transceiver_entry.split(mibs.TABLE_NAME_SEPARATOR_VBAR)[-1]
-            self._update_transceiver_cache(interface)
-
-    def _update_per_namespace_data(self, pubsub):
-        """
-        Update cache.
-        Here we listen to changes in STATE_DB TRANSCEIVER_INFO table
-        and update data only when there is a change (SET, DELETE)
-        """
-
-        # This code is not executed in unit test, since mockredis
-        # does not support pubsub
-        while True:
-            msg = pubsub.get_message()
-
-            if not msg:
-                break
-
-            transceiver_entry = msg["channel"].split(":")[-1]
-            data = msg['data'] # event data
-
-            # extract interface name
-            interface = transceiver_entry.split(mibs.TABLE_NAME_SEPARATOR_VBAR)[-1]
-
-            # get interface from interface name
-            ifindex = port_util.get_index_from_str(interface)
-
-            if ifindex is None:
-                # interface name invalid, skip this entry
-                mibs.logger.warning(
-                    "Invalid interface name in {} \
-                     in STATE_DB, skipping".format(transceiver_entry))
-                continue
-
-            if "set" in data:
-                self._update_transceiver_cache(interface)
-            elif "del" in data:
-                # remove deleted transceiver
-                remove_sub_ids = [mibs.get_transceiver_sub_id(ifindex)]
-
-                # remove all sensor OIDs associated with removed transceiver
-                for sensor in SENSOR_NAME_MAP:
-                    remove_sub_ids.append(mibs.get_transceiver_sensor_sub_id(ifindex, sensor))
-
-                for sub_id in remove_sub_ids:
-                    if sub_id and sub_id in self.physical_entities:
-                        self.physical_entities.remove(sub_id)
+        self.physical_name_map[chassis_sub_id] = self.CHASSIS_NAME
+        self.physical_description_map[chassis_sub_id] = self.CHASSIS_NAME
+        self.physical_contained_in_map[chassis_sub_id] = 0
+        self.physical_fru_map[chassis_sub_id] = self.NOT_REPLACEABLE
+
+        # Add a chassis mgmt node
+        chassis_mgmt_sub_id = (mibs.CHASSIS_MGMT_SUB_ID,)
+        self.add_sub_id(chassis_mgmt_sub_id)
+        self.physical_classes_map[chassis_mgmt_sub_id] = PhysicalClass.CPU
+        self.physical_contained_in_map[chassis_mgmt_sub_id] = mibs.CHASSIS_SUB_ID
+        self.physical_parent_relative_pos_map[chassis_mgmt_sub_id] = 1
+        name = 'MGMT'
+        self.physical_description_map[chassis_mgmt_sub_id] = name
+        self.physical_name_map[chassis_mgmt_sub_id] = name
+        self.physical_fru_map[chassis_mgmt_sub_id] = self.NOT_REPLACEABLE
```

Comment:
I see most contents are static, why we need to `reinit_data()` every time?

Reply:
Because we clear all existing data before reinit_data(), see line 304~316. And in those data, we have both static/dynamic data, it is hard to tell which one is static or dynamic unless we hardcode it or add a tag for each data(which is even complicated). So I choose to reinit these static content every time. Any suggestion?